docs/guests.md

```diff
@@ -53,6 +53,27 @@ rc-service xe-guest-utilities start
 apt install xe-guest-utilities
 ```
 
+#### openSUSE Leap 15.2 with transactional-updates
```

Comment:
The section you put this guide in is named "### Install from the distro's online repositories" but your guide is about overcoming issues when installing the tools from the ISO. We need to find a different place for it. I suggest you create a `#### Specific cases` section after `#### Other Linux distributions`, under which you'd then put your `##### openSUSE Leap 15.2 with transactional-updates`  guide.

Reply:
Ups, my bad seems like i mixed it up in the edit. Wanted to put it under ### Install from the guest tools ISO and than #### Other Linux distributions. Scince it is installed via the ISO image.

Reply:
Since the "other" section is small and generic, I prefer to have yours after. It will still belong to ### Install from the guest tools ISO. I'll do the change, fix a few typos, and merge.